src/main/java/com/dpgil/pathlinker/path_linker/internal/PathLinkerModel.java

```diff
@@ -59,6 +59,8 @@
 	private HashSet<CyEdge> superEdges;
 	/** Number of shared nodes between sources and targets */
 	private int commonSourcesTargets;
+	/** Whether or not to treat all paths with same weight as one "k" path */
+	private boolean includePathScoreTies;
```

Comment:
This comment is how you originally implemented the option. Update it with a better description